config/initializers/datadog.rb

```diff
@@ -30,5 +30,36 @@
     # Enable ASM
     c.appsec.enabled = true
     c.appsec.instrument :rails
+
+    # Add dogstatsd_mapper_profiles field
+    c.dogstatsd_mapper_profiles = [
+      {
+        name: 'sidekiq',
+        prefix: 'sidekiq.',
+        mappings: [
+          {
+            match: 'sidekiq.sidekiq.*',
+            match_type: 'regex',
+            name: 'sidekiq.$1'
+          },
+          {
+            match: 'sidekiq.jobs.*perform',
```

Comment:
Shouldn't this be `sidekiq.jobs..*perform`?

Reply:
No. the dot there is any character. It's not necessary.

Reply:
I just noticed it was there in DD's docs.

Reply:
Yeah, I had to modify it. Adding the extra `.` would be kinda redundant.